pkgs/by-name/rd/rdesktop/package.nix

```diff
@@ -48,6 +49,13 @@ stdenv.mkDerivation (rec {
     "--disable-smartcard"
   ] ++ lib.optional (!enableCredssp) "--disable-credssp";
 
+  patches = [
+    (fetchpatch {
+      url = "https://github.com/rdesktop/rdesktop/commit/53ba87dc174175e98332e22355ad8662c02880d6.patch";
+      hash = "sha256-ORGHdabSu9kVkNovweqFVS53dx6NbiryPlgi6Qp83BA=";
+    })
+  ];
```

Comment:
thanks for adding the patch.

looking over https://src.fedoraproject.org/rpms/rdesktop/tree/rawhide I think there are a couple more that are relevant.

https://src.fedoraproject.org/rpms/rdesktop/blob/rawhide/f/rdesktop-configure-c99.patch
fixes a config check from failing due to implicit function definition.

https://src.fedoraproject.org/rpms/rdesktop/blob/rawhide/f/394.patch
reorders a null check to avoid dereferencing a null pointer.
also available: https://github.com/rdesktop/rdesktop/commit/105c8cb69facf26238cd48f14ca9dbc0ff6be6bd

I don't have any knowledge on the final one.

Reply:
thanks! looks good.